src/Gadfly.jl

```diff
@@ -388,14 +388,14 @@ function render_prepare(plot::Plot)
     # they are missing.
     datas = Array{Data}(undef, length(plot.layers))
     for (i, layer) in enumerate(plot.layers)
-        if layer.data_source === nothing && isempty(layer.mapping)
+        if isnothing(layer.data_source) && isempty(layer.mapping)
```

Comment:
I'd be willing to bet `=== nothing` is more efficient. The compiler loves `===`.

Reply:
Simply changing occurrences of `== nothing` to use `===` (and likewise for the negative) should provide a nice improvement.

Reply:
For reference, this issue applies: https://github.com/JuliaLang/julia/issues/27681

Reply:
Good to know. I'll update with straight `===` and see if it makes a difference.

Reply:
That would also resolve the `isnothing` name qualification discussion.

Reply:
I've pushed a commit changing from `isnothing` to `=== nothing`. Still need to get back to the computer I've benchmarked on before to see if it makes a difference.